kotlin-styled-next/build.gradle.kts

```diff
@@ -33,22 +35,27 @@ dependencies {
 }
 
 val printBenchmarkResults by tasks.registering {
-    doLast {
-        // implicit dependency to benchmark classes names
-        val fileNames = listOf(
-            "AddStyledElements",
-            "ConvertToStyled",
-            "CssBuildersInject",
-            "AddDuplicateCss",
-            "DataTypeOperations",
-            "InjectCssNPlusOne"
-        )
-        fileNames.forEach { test ->
-            val report = layout.buildDirectory
-                .file("reports/tests/test/classes/benchmark.$test.html")
-                .get().asFile.readText()
+    // implicit dependency to benchmark classes names
```

Comment:
Could we move this change in separate PR?

Reply:
Sure thing https://github.com/JetBrains/kotlin-wrappers/pull/2380